backend/lib/database/postgres/census_tiger.sh

```diff
@@ -0,0 +1,52 @@
+# NOTE - Running this file requires postgres AND postigs.
```

Comment:
Please add a comment saying where this should be run.

Reply:
add it to the readme?